.env.example

```diff
@@ -0,0 +1 @@
+# Add env variables here
```

Comment:
How is this file working?

Reply:
I couldn't see any place from where we are fetching environment variables

Reply:
Yes not currently, but will be required later